Pipfile.lock

```diff
@@ -1,7 +1,7 @@
 {
     "_meta": {
         "hash": {
-            "sha256": "0b5ba84c98d9ff3e115f9a26ceb3950ec4911f0812de157995f51e60b8963959"
+            "sha256": "50dc27996f4edac69cb31d2c395863b0ccd06816711cce083393b190725fe83d"
```

Comment:
Should this be merged?

Reply:
I think so, it makes the enviroment more similar to all of us.